src/routes/webhooks/paddle.ts

```diff
@@ -74,6 +82,39 @@ const updateUserSubscription = async ({
     );
     return false;
   }
+
+  const { gifter_id: gifterId, duration } = customData;
+  const durationTime = duration && parseInt(duration);
```

Comment:
is this part set as custom data on frontend side?

Reply:
Not yet, but we need to add it during the next tasks

Reply:
in that case it can be mended by malicious user so we should have it coded on the API side only and not be dynamic and user injectable

Reply:
Ok, since currently we plan only for yearly plan gift I can hardcode it here ✔️  @rebelchris @capJavert

Reply:
Updated ✔️